types/vote_test.go

```diff
@@ -130,12 +130,17 @@ func TestVoteSignBytesTestVectors(t *testing.T) {
 		},
 		// containing vote extension
 		5: {
-			"test_chain_id", &Vote{Height: 1, Round: 1, VoteExtension: VoteExtension{
-				AppDataToSign:             []byte("signed"),
-				AppDataSelfAuthenticating: []byte("auth"),
-			}},
+			"test_chain_id", &Vote{
+				Type:           0,
+				Height:         1,
+				Round:          1,
+				BlockID:        BlockID{},
+				Timestamp:      time.Time{},
+				ValidatorIndex: 0,
```

Comment:
It would be fine to omit the zero-valued initializers here.

```suggestion
				Height:         1,
				Round:          1,
```

Reply:
Sorry, poor IDE defaults again here. Will fix that ASAP :+1: